backend/modules/algoProviders/algoProvidersManager.py

```diff
@@ -26,20 +28,29 @@ def setup_algo_providers():
         if url[-1] == "/":
             url = url[:-1]
 
-        print(" - Adding AlgoProviders " + name + " from " + url + " - ")
+        print(
+            " - Adding AlgoProvider "
+            + colored(name, DEBUG_COLOR)
+            + " ("
+            + colored(url, DEBUG_COLOR)
+            + ")"
+        )
         try:
             algo_provider = AlgoProvider(url, name)
+            algo_providers.append(algo_provider)
```

Comment:
Are you sure you wanna add to the list even if it is not accessible ?

Reply:
If the data-provider is added while being not accessible, it will appear in DebiAI but will be displayed as `not available` so nothing wrong will append, it will serve as debug info for the user.

Note that there is no changes from last version, I've just moved the line on top of the `if`.